cli.py

```diff
@@ -66,8 +66,20 @@ def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delet
         return
 
     if scrape:
-        if dm is None:
+        if not dm:
```

Comment:
Checking for Boolean value is not the same as checking for None-ness. This is particularly true in Python as None in python is very specific, whereas in language like C++ "Null" is basically equivalent to "0".